web-app/src/i18n/pl/common.ts

```diff
@@ -3,4 +3,5 @@ export default {
   cancel: 'Anuluj',
   create: 'Stwórz',
   name: 'Nazwa',
+  noResults: 'Brak wyników wyszukiwania'
```

Comment:
NoResults != NoSearchResults
So is this a search result or some other results?
This is a potential name for error because this is a common locale.